<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationFunButtonWidget</class>
 <widget class="QWidget" name="StationFunButtonWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1482</width>
    <height>58</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>44</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>58</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>进路建立</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>总取消</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_3">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>信号重开</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_4">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">:disabled{color:gray;}
:enabled {color:red;}</string>
     </property>
     <property name="text">
      <string>引导按钮</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_5">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
:disabled{color:gray;}
:enabled {color:red;}</string>
     </property>
     <property name="text">
      <string>引导总锁</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_6">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
:disabled{color:gray;}
:enabled {color:red;}</string>
     </property>
     <property name="text">
      <string>总人解</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_7">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">:disabled{color:gray;}
:enabled {color:red;}</string>
     </property>
     <property name="text">
      <string>区故解</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_8">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>总定位</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_9">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>总反位</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_10">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>单锁</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_11">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>单解</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_12">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>封锁</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_13">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>解封</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_14">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
:disabled{color:gray;}
:enabled {color:red;}</string>
     </property>
     <property name="text">
      <string>功能按钮</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_15">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
:disabled{color:gray;}
:enabled {color:red;}</string>
     </property>
     <property name="text">
      <string>坡道解锁</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_16">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
:disabled{color:gray;}
:enabled {color:red;}
</string>
     </property>
     <property name="text">
      <string>分路不良</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_17">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>确认空闲</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_18">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>点灯</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_19">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
:disabled{color:gray;}
:enabled {color:red;}</string>
     </property>
     <property name="text">
      <string>灭灯</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_20">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>辅助菜单</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_21">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>命令清除</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_22">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>命令下达</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_23">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>状态选择</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_24">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>方式转换</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
